tests/bears/BearTest.py

```diff
@@ -467,6 +467,28 @@ def test_get_non_optional_settings(self):
                          {'y': ('Second value, but better.', int),
                           'w': ('Fourth value.', float)})
 
+    def test_no_warning_debug_enabled_LocalBear(self):
+        self.settings.append(Setting('log_level', 'DEBUG'))
+        self.uut = LocalBear(self.settings, self.queue)
+        self.uut.execute('filename.py', 'file\n')
+        self.check_message(LOG_LEVEL.DEBUG)
+        # Fails because of no run() implementation
+        self.assertNotEqual(LOG_LEVEL.ERROR,
```

Comment:
This is a no-op.  `LOG_LEVEL.ERROR` is not equal to the string below.  It doesnt verify anything.  Refer to other tests which inspect `logging` output.